index.js

```diff
@@ -1,54 +1,104 @@
-const authorInput = document.querySelector('#author');
-const bookList = document.querySelector('#book-list');
-const formGroup = document.querySelector('#form-container');
-let addOn;
-let bookObj = JSON.parse(localStorage.getItem('saveItem')) || [];
-
-// Add book
-function newAdd() {
-  addOn = {
-    title: titleInput.value,
-    author: authorInput.value,
-    id: Math.floor(Math.random() * 1000000),
-  };
-  bookObj.push(addOn);
-  localStorage.setItem('saveItem', JSON.stringify(bookObj));
-}
+// Add books classas
+class DisplayOn {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
 
-// Remove
-function removeBook(id) {
-  bookObj = bookObj.filter((filterObj) => filterObj.id !== id);
-  localStorage.setItem('saveItem', JSON.stringify(bookObj));
-}
+  static displayBooks() {
+    const books = DisplayOn.getBooks();
 
-// Create Books
-function addBooks(addOn) {
-  const tableDiv = document.createElement('div');
-  const bookTitle = document.createElement('h3');
-  const authorName = document.createElement('h3');
-  const removeBtn = document.createElement('button');
-  const line = document.createElement('hr');
-  bookTitle.innerText = addOn.title;
-  authorName.innerText = addOn.author;
-  removeBtn.innerText = 'Remove';
-  tableDiv.append(bookTitle, authorName, removeBtn, line);
-  bookList.appendChild(tableDiv);
-  removeBtn.addEventListener('click', () => {
-    removeBtn.parentElement.remove();
-    removeBook(addOn.id);
-  });
+    books.forEach((bookAddOn) => DisplayOn.addToBooks(bookAddOn));
+  }
+
+  static addToBooks(bookAddOn) {
+    const bookList = document.querySelector('#book-list');
+    const tableDiv = document.createElement('div');
+    tableDiv.classList.add('collection');
+    tableDiv.innerHTML = `
+    <div class="collection-text">
+    <p>"${bookAddOn.title}"</p>
+    <p>by</p>
+    <p>${bookAddOn.author}</p>
+    </div>
+    <button type="submit" class="delete">Remove</button>
+    `;
+    bookList.appendChild(tableDiv);
+  }
+
+  static deleteBook(el) {
+    if (el.classList.contains('delete')) {
+      el.parentElement.remove();
+    }
+  }
+
+  static clearInputFields() {
+    document.querySelector('#title').value = '';
+    document.querySelector('#author').value = '';
+  }
+
+  // Store class local storage
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
+
+  static saveBooks(book) {
+    const books = DisplayOn.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removeBooks(title) {
+    const bookList = document.querySelector('#book-list');
+    let books = DisplayOn.getBooks();
+    const newBookObj = books.filter((book) => book.title !== title);
+    localStorage.setItem('books', JSON.stringify(newBookObj));
```

Comment:
- [ ] The same issue that was pointed out by the previous reviewer is still recurring. When I add two books of the same title and delete one, both books are deleted from the local storage and the page. But only the deleted book should have been removed. You can resolve this by attaching a kind of unique `id` to your books and implementing your book deletion using the `id`, instead of `title`.

  I added books of the same title | I deleted one but both are gone
  :---:|:---:
  ![all-e](https://user-images.githubusercontent.com/81468675/191532367-4c7bb39f-aa47-427c-8536-6c735466d5ee.jpg) | ![all-e-2](https://user-images.githubusercontent.com/81468675/191532362-aa383efb-1681-4cc2-b5df-3de070fefcdc.jpg)